Makefile

```
@@ -259,6 +259,13 @@ generate-go-conversions: $(CONVERSION_GEN) ## Run all generate-go-conversions
 		--output-file-base=zz_generated.conversion \
 		$(CONVERSION_GEN_OUTPUT_BASE) \
 		--go-header-file=./hack/boilerplate/boilerplate.generatego.txt
+		$(MAKE) clean-generated-conversions SRC_DIRS="./api/v1beta1"
+		$(CONVERSION_GEN) \
+		--input-dirs=./api/v1beta1 \
+		--build-tag=ignore_autogenerated_core \
+		--output-file-base=zz_generated.conversion \
+		$(CONVERSION_GEN_OUTPUT_BASE) \
+		--go-header-file=./hack/boilerplate/boilerplate.generatego.txt			
```

any reason to separate these commands instead of running all at once ?

Something around this may work:
```suggestion
		$(MAKE) clean-generated-conversions SRC_DIRS="./api/v1alpha1 ./api/v1beta1"
		$(CONVERSION_GEN) \
		--input-dirs=./api/v1alpha1 \
		--input-dirs=./api/v1beta1 \
		--build-tag=ignore_autogenerated_core \
		--output-file-base=zz_generated.conversion \
		$(CONVERSION_GEN_OUTPUT_BASE) \
		--go-header-file=./hack/boilerplate/boilerplate.generatego.txt			
```

---

done in https://github.com/netobserv/network-observability-operator/pull/329/commits/ae2fabd82a3b4011bf18925c3fbe6d66a9853989